src/config/env.ts

```diff
@@ -7,11 +7,25 @@ const env = dotenv.config({
   path: path.resolve(process.cwd(), filename),
 });
 
+function getProxyFromEnvironment(): string{
+  let proxyHost: string = process.env.EXTERNAL_ONLY_PROXY_HOST;
+  let proxyPort: string = process.env.EXTERNAL_ONLY_PROXY_PORT;
+
+  if(!proxyHost || !proxyPort){
+    return '';
+  }
+  const proxyAddress = `http://${proxyHost}:${proxyPort}`;
```

Comment:
Shouldn't this be https?

Reply:
no, our proxy is explicitly available via `http` (but the environment variable doesn't include the prefix, so I had to work around that like this)